.github/CODEOWNERS

```diff
@@ -315,4 +315,5 @@
 /templates/**/workflow @kimizhu @dooriya @swatDong
 /templates/constraints/yml/actions @hund030 @eriolchan @huimiu
 /templates/python/custom-copilot-basic @frankqianms @blackchoey
+/templates/python/custom-copilot-rag-azure-ai-search @frankqianms @blackchoey
```

Comment:
let's talk with Wei whether we can be codeowner of `/templates/python/*`, this should reduce some mantainance work to this file

Reply:
I remember there will be an api agent python version template, which owned by @KennethBWSong and @SLdragon. 
Am i right? if so, then we can not be codeowner of `/templates/python/*`

Reply:
Could assign that particular template to them by adding an extra line, it will override the owner of `templates/python/*`. I just don't expect we need to add code owner for every python template.
Not a big deal. Let's merge this PR first.

Reply:
done